tests/db_functions/datetime/test_extract_trunc.py

```diff
@@ -1935,3 +1935,113 @@ def test_trunc_invalid_field_with_timezone(self):
             DTModel.objects.annotate(
                 hour_melb=Trunc("start_time", "hour", tzinfo=melb),
             ).get()
+
+    def test_trunc_with_etc_offset_timezone(self):
+        start_datetime = datetime(2015, 6, 15, 14, 30, 50, 321)
+        end_datetime = datetime(2016, 6, 15, 14, 10, 50, 123)
+        start_datetime = timezone.make_aware(start_datetime)
+        end_datetime = timezone.make_aware(end_datetime)
+        self.create_model(start_datetime, end_datetime)
+        self.create_model(end_datetime, start_datetime)
+        etc_timezone = zoneinfo.ZoneInfo("Etc/GMT+10")
+
+        def test_datetime_kind(kind):
```

Comment:
Calling these helper functions `assert_` is a bit easier to understand I think.

```suggestion
        def assert_datetime_kind(kind):
```

Reply:
We follow the camel case naming used by assertions in `unittest`, so `assertDatetimeKind()`.